public/js/main.js

```diff
@@ -69,19 +67,41 @@ todoInput.onkeypress = function (e){
 function deleteTask(taskId){
   var index;
   for(let i=0;i<allList.length;i++){
-    if(allList[i].id === taskId) index = i; 
+    if(allList[i].id === taskId) {
```

Comment:
Can we name the `allList` to `todoList`? Should add the domain in the variable name

- [ ] consider